ext/opentelemetry-ext-dbapi/src/opentelemetry/ext/dbapi/__init__.py

```diff
@@ -125,6 +125,15 @@ def wrap_connect_(
         logger.warning("Failed to integrate with DB API. %s", str(ex))
 
 
+def unwrap_connect(
```

Comment:
This is going to be there for multiple instrumentors maybe adding this as utility in opentelemetry-auto-instrumentation would be make sense

Reply:
I'm not sure I got your comment.

The `unwrap_connect` is related to dbapi so it's here and available to every db library that wants to use it. The internal logic on that method is to unwrap an object, I think we should create generic wrap, unwrap functions in a module available to all integrations. @codeboten already faced a similar issue in other instrumentors.

Reply:
Yes sorry, I was referring to a generic wrap/unwrap like you mentioned shared between integrations

Reply:
We have an issue to track this https://github.com/open-telemetry/opentelemetry-python/issues/610.